README.md

```diff
@@ -1,117 +1,104 @@
-# helia-service-worker-gateway
+<h1 align="center">
+  <br>
+  <img src="https://github.com/ipfs-shipyard/service-worker-gateway/assets/157609/4931e739-a899-4b18-91f2-2a2bcafb5c33" alt="logo" title="logo" width="200"></a>
+  <br>
+  Service Worker IPFS Gateway
+  <br>
+</h1>
+
+<p align="center" style="font-size: 1.2rem;">Decentralizing IPFS Gateways by running hash verification on end user's machine.</p>
+
+<p align="center">
+  <a href="https://ipfs.tech"><img src="https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square" alt="Official Part of IPFS Project"></a>
+  <a href="https://discuss.ipfs.tech"><img alt="Discourse Forum" src="https://img.shields.io/discourse/posts?server=https%3A%2F%2Fdiscuss.ipfs.tech"></a>
+  <a href="https://matrix.to/#/#ipfs-space:ipfs.io"><img alt="Matrix" src="https://img.shields.io/matrix/ipfs-space%3Aipfs.io?server_fqdn=matrix.org"></a>
+  <a href="https://github.com/ipfs-shipyard/service-worker-gateway/actions"><img src="https://img.shields.io/github/actions/workflow/status/ipfs-shipyard/service-worker-gateway/main.yml?branch=main" alt="ci"></a>
+  <a href="https://github.com/ipfs-shipyard/service-worker-gateway/releases"><img alt="GitHub release" src="https://img.shields.io/github/v/release/ipfs-shipyard/service-worker-gateway?filter=!*rc*"></a>
+</p>
+
+<hr />
+
+## About
+
+This project demonstrates
+the use of [Helia](https://helia.io) (IPFS implementation in JS)
```

Comment:
I'd link to the repo which has less broken links and more context. 
```suggestion
the use of [Helia](https://github.com/ipfs/helia) (IPFS implementation in JS)
```

Reply:
I think we should solve the problems that exist at helia.io and use that link in documentation, but maybe we can use an .fyi link so we can change it in the future?